packages/core/src/UI/Components/ThemeController/Lib/UseFlyonUIThemeFont.ts

```
@@ -0,0 +1,72 @@
+import type { FlyonUITheme, FlyonUIThemeFont } from '@/UI/Components';
+import type { MaybeRefOrGetter }               from 'vue';
```

ESLint

[error] Expected "vue" (type) to come before "@/UI/Components" (internal-type).

(perfectionist/sort-imports)